SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+Please see our [Support Policy](README.md#support-policy)
+
+## Reporting a Vulnerability
+
+If you have found a security issue, please contact the security team directly at security@datadoghq.com.
```

Comment:
Does this apply specifically to vulnerabilities in dd-trace-go itself? As opposed to possible vulnerabilities in our dependencies?

Reply:
OOooooo Very good point. We should clarify this is only for vulnerabilities in dd-trace-go itself NOT in dependencies

Reply:
I went ahead and clarified that. Take another look 👍